app/presenters/field/compound_presenter.rb

```diff
@@ -0,0 +1,29 @@
+class Field::CompoundPresenter < FieldPresenter
+  def input(form, method, options = {})
+    form.text_field(method, input_defaults(options).reverse_merge(:help => help, :value => field.template, :readonly => true))
+  end
+
+  def value
+    render_template
+  end
+
+  def render_template
+    tpl = JSON.parse(field.template)
+
+    local_tpl = tpl[I18n.locale.to_s] || ''
```

Comment:
plutot que 2 variables, dont 1 qui sert que 1 fois et l'autre qui contient un nom abrégé:

```suggestion
    template = JSON.parse(field.template)[I18n.locale.to_s] || ''
```

(tpl ca veut bien dire template ??)

Reply:
yes repris de le exemple present dans catima remplacé par local_template